package.json

```diff
@@ -9,8 +9,8 @@
     "dev": "babel -w src -d lib",
     "lint": "eslint src test",
     "fix": "eslint src test --fix",
-    "test": "mocha -t 5000",
-    "ci": "npm run lint && cross-env DEBUG_CALLS=true npm run test",
+    "test": "mocha -t 20000",
+    "ci": "npm run lint && cross-env npm run test",
```

Comment:
Had to increase timeout for appveyor tests to pass (see main conversation thread for error there). Also removed DEBUG_CALLS to make test output readable. If you want it restored, happy to do as part of this PR